includes/class-wcmp-export.php

```diff
@@ -523,7 +523,8 @@ public function prepare_return_shipment_data($order_id, $options) {
                 'only_recipient',
                 'signature',
                 'return',
-                'large_format'
+                'large_format',
+                'age_check'
```

Comment:
```suggestion
                'age_check',
```

Reply:
Op het einde van de array is het niet noodzakelijk om een komma te plaatsen.